draft-ietf-quic-recovery.md

```diff
@@ -130,15 +145,17 @@ number for transmissions, and any data that is to be delivered to the receiving
 application(s) is sent in one or more streams, with delivery order determined
 by stream offsets encoded within STREAM frames.
 
-QUIC's packet number is strictly increasing, and directly encodes transmission
-order.  A higher QUIC packet number signifies that the packet was sent later,
-and a lower QUIC packet number signifies that the packet was sent earlier.  When
-a packet containing frames is deemed lost, QUIC rebundles necessary frames in a
-new packet with a new packet number, removing ambiguity about which packet is
-acknowledged when an ACK is received.  Consequently, more accurate RTT
-measurements can be made, spurious retransmissions are trivially detected, and
-mechanisms such as Fast Retransmit can be applied universally, based only on
-packet number.
+QUIC's packet number is strictly increasing within a packet number
+space, and directly encodes transmission order.  A higher QUIC packet
+number signifies that the packet was sent later, and a lower QUIC
+packet number signifies that the packet was sent earlier.  When a
+packet containing frames is deemed lost, QUIC rebundles the
+retransmittable data in a new packet with a new packet number,
+removing ambiguity about which packet is acknowledged when an ACK is
+received.  Consequently, more accurate RTT measurements can be made,
+spurious retransmissions are trivially detected, and mechanisms such
+as Fast Retransmit can be applied universally, based only on packet
+number.
```

Comment:
The only change here seems to be that it was rewrapped to a smaller width than the rest of the document.  Is this necessary?